### PR TITLE
document connecting to other instances

### PR DESCRIPTION
**Pull request by jjcarstens**
As requested in [NervesHubCLI](https://github.com/nerves-hub/nerves_hub_cli/pull/163#issuecomment-733748072) and resolves https://github.com/nerves-hub/nerves_hub_link/issues/58

### CODE REVIEW

File: nerves-hub/setup/connecting-other-envs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Connecting to other environments
+
+Some cases may require you to target other instances of NervesHub, such as testing a staging environment or directing devices to a self-hosted NervesHub. In these cases, you'll need to specify a few additional details in your application config:
```
Comment (fhunleth) [inline nerves-hub/setup/connecting-other-envs.md:3]:
```suggestion
Some cases may require you to target other instances of NervesHub, such as testing a staging environment or directing devices to a self-hosted NervesHub. In these cases, you'll need to add your server's information to your project's application config:
```

**Review by fhunleth — APPROVED** [review]
I made a couple super-minor changes. This looks good to me overall. Thanks for adding it.

File: nerves-hub/setup/README.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 ## Getting Started
 
-The following sections will walk you through updating your Nerves project to work with the [nerves-hub.org](https://nerves-hub.org) NervesHub server. Using your own NervesHub server will require setting URLs to point elsewhere and is not covered below to simplify first usage.
+The following sections will walk you through updating your Nerves project to work with the [nerves-hub.org](https://nerves-hub.org) NervesHub server. For connecting to your own NervesHub or another location, look at [`Setup: Connecting to other environemnts`](connecting-other-envs.md) first, then continue from here.
```
[the remark this answers is not in the document]
Comment (jjcarstens) [reply nerves-hub/setup/README.md:5]: I'm failing to see what was actually changed in this suggestion? They look the same to me?

[the remark this answers is not in the document]
Comment (fhunleth) [reply nerves-hub/setup/README.md:5]: This fixes a spelling error: environemnts -> environments.

[the remark this answers is not in the document]
Comment (jjcarstens) [reply nerves-hub/setup/README.md]: 🤦‍♂️

**Review by fhunleth — APPROVED** [review]

**Pull request merged**